site/content/docs/04-preloading.md

```diff
@@ -2,7 +2,15 @@
 title: Preloading
 ---
 
-As seen in the [routing](docs#Routing) section, page components can have an optional `preload` function that will load some data that the page depends on. This is similar to `getInitialProps` in Next.js or `asyncData` in Nuxt.js.
+Page components can define a `preload` function that runs before the component is created. The values it returns are passed as props to the page.
+
+Preload functions are typically used to load data that the page depends on, hence its name. Having the data available before the page load allows you to avoid flashing while it is fetched.
```

Comment:
"Having the data available before the page load allows you to avoid flashing while it is fetched." - I'm not sure that it would necessarily cause flashing to fetch data on the client. Many sites load some amount of data after the client has loaded and just have a portion of the page that's not filled in or has a loading widget or something. I might just remove this sentance

Reply:
Yeah, this is from trying to preserve the sentence "it runs before the component is created, allowing you to avoid flashes while data is fetched." But you're right; "flashing" doesn't cover it. If anything, it's things jumping around on the page. I'll rewrite it.